lxd/instance_rebuild.go

```diff
@@ -99,7 +99,7 @@ func instanceRebuildPost(d *Daemon, r *http.Request) response.Response {
 
 		if req.Source.Type != "none" {
 			sourceImage, err = getSourceImageFromInstanceSource(ctx, s, tx, targetProject.Name, req.Source, &sourceImageRef, dbInst.Type.String())
-			if err != nil {
+			if err != nil && !api.StatusErrorCheck(err, http.StatusNotFound) {
```

Comment:
Why do we want to ignore image not found errors here?

Reply:
Because otherwise we end up being limited to only images we already have in our cache.
We need to keep going with no sourceImage so that LXD will perform the needed download.

Basically if you currently do `lxc rebuild images:archlinux foo` and you've never used `images:archlinux` before, the command will fail with `Image not found` rather than go and download it.